stable/kommander/requirements.lock

```diff
@@ -1,10 +1,10 @@
 dependencies:
 - name: kubeaddons-catalog
   repository: https://mesosphere.github.io/charts/staging
-  version: 0.1.5
+  version: 0.1.6
 - name: kommander-cluster-lifecycle
   repository: https://mesosphere.github.io/kommander-cluster-lifecycle/charts
-  version: v0.4.6
+  version: 0.4.6
```

Comment:
the tooling automated this, I didn't mean to touch it?

Reply:
i think others have been seeing that too

Reply:
I'm on the latest `v2.16.3` client, I wonder if that has anything to do with it :thinking: